flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/nodes/exec/ExecNodeContext.java

```diff
@@ -167,12 +170,19 @@ public ExecNodeContext withId(int id) {
      */
     @JsonValue
     public String getTypeAsString() {
+        if (name == null || version == null) {
+            throw new TableException(
+                    String.format(
+                            "Can not serialize ExecNode with id: %d. Missing type, this is a bug,"
+                                    + " please file a ticket.",
```

Comment:
I suggest rephrasing to raising a Jira and including the url.

Reply:
The current phrasing is in line with other similar places. I would also not use a proprietary system name in the code. It's not said Apache and Flink in particular continues using JIRA.